content/docs/user-guide/experiment-management/cleaning-experiments.md

```diff
@@ -210,16 +210,16 @@ Removed experiments: exp-bb09c
 
 ## Removing queued experiments
 
-When you've created experiments to be run in the queue with
-`dvc exp run --queue` and later decide not to run them, you can remove them with
-`dvc exp remove --queue`.
+When you've created experiments to be run in the queue with `dvc queue start`
+and later decide not to run them, you can remove them with
+`dvc queue remove --queued`.
```

Comment:
@pmrowla should we make this update in other places? The docs still mention `exp run --queue` a lot.

Reply:
The issue here was that `dvc exp run --queue` was explaining the way the user "created experiments with ..." but `dvc queue start` is the way the user would "run experiments with ...".

So the original intention was more like

> When you've created experiments with `dvc exp run --queue` and later decide not to run them, ...

I guess the question is whether or not we can just say "added experiments to the queue" or "queued experiments" instead of providing the commands in these parts of the user guide?

Reply:
> the question is whether or not we can just say "added experiments to the queue" or "queued experiments" instead of providing the commands

Great Q @pmrowla. I think that in the User Guide (as is the case), we should indeed use descriptive phrasing more, but link to the corresponding sections in the same or other guides.

> However, in Refs it's better to link to other commands so it stays self-contained (linking to guides additionally in admonitions is also good, for those who want/need more reading).